tcl/org.testeditor.tcl.dsl.tests/src/org/testeditor/tcl/dsl/tests/formatter/TclModelFormatterTest.xtend

```diff
@@ -0,0 +1,75 @@
+package org.testeditor.tcl.dsl.tests.formatter
+
+import org.junit.Test
+
+class TclModelFormatterTest extends AbstractTclFormatterTest {
```

Comment:
splitted into two, several constructs are checked by the respective tml ...FormatterTest